RubeusCore/Include/logger_component.h

```diff
@@ -68,7 +76,7 @@
 // In case the build configuration is not "Debug"
 
 // Deprecated for non-debug builds
-#define LOG(x) std::cout << x << std::endl;
+#define LOG(x) std::cout  << "\033[1;" << ColorCoder::foregroundColorMap["yellow"] << "m" << x << "\033[0m" << std::endl;
```

Comment:
Any explanation on the `\033[1` and `\033[0m`?

Reply:
I was following this to enable colored output-
https://stackoverflow.com/questions/2616906/how-do-i-output-coloured-text-to-a-linux-terminal